Source/Scene/WebMapTileServiceImageryProvider.js

```diff
@@ -316,8 +316,17 @@ function WebMapTileServiceImageryProvider(options) {
     Rectangle.northeast(this._rectangle),
     this._minimumLevel
   );
+
```

Comment:
If you can be bothered, you should PR just this change to Cesium. It's independent of our getFeatureInfo stuff.

Reply:
It is possible that this is needed because we aren't provided valid tile matrix sets - i will revisit this when I fix a few WMTS issues